nixos/modules/services/x11/desktop-managers/gnome3.nix

```diff
@@ -409,9 +409,7 @@ in
         baobab
```

Comment:
we need to add reverting https://github.com/NixOS/nixpkgs/commit/70dc25abd9a63ca3839c08bba8cba9e7bc974b04 also

Reply:
Done!